compiler/noirc_driver/src/lib.rs

```diff
@@ -544,14 +544,15 @@ pub fn compile_no_check(
         return Ok(cached_program.expect("cache must exist for hashes to match"));
     }
     let return_visibility = program.return_visibility;
+    let ssa_evaluator_options = noirc_evaluator::ssa::SsaEvaluatorOptions {
+        enable_ssa_logging: options.show_ssa,
+        enable_brillig_logging: options.show_brillig,
+        force_brillig_output: options.force_brillig,
+        print_codegen_timings: options.benchmark_codegen,
```

Comment:
💭 I'm not sure which final names should we use here ("enable_ssa_logging" vs. "show_ssa" vs. "print_ssa_passes"). Maybe using all names would be better, but then using different names at different layers might also make sense.